Clean Up Experiments Folder

jlopez8 opened the pull request:
Clean up experiments directory.

rkingery reviewed (changes requested):
Notes:

- Update your gitignore to reflect the new structure. For example, you still have things in there that reference the old `experiments` folder structure.
- Remove the following hidden files and folders (with all contents contained) that cookie cutter created: `.github`, `.editorconfig`, `.travis.yml`
- For your *old* experiments stuff, keep them *in* the repo, but only local. Looks like what you did was moved it all out and just copied the files you wanted to show back into `experiments`. Do it the way I said during the meeting. Name your *old* `experiments` folder something else (e.g. `local`), add that to the gitignore, and then use the *new* `experiments` folder to store the specified outputs. You don't want *anything* project-related sitting outside your repo, even if it's just local files that github will never see. Keep them all in one place.
- It also wouldn't be a bad idea to rename the branch here to something more appropriate, given the name isn't exactly reflective of the fact that this branch is really now cleaning up the repo for later development.

jlopez8 commented:
> Notes:
> 
>     * Update your gitignore to reflect the new structure. For example, you still have things in there that reference the old `experiments` folder structure.
> 
>     * Remove the following hidden files and folders (with all contents contained) that cookie cutter created: `.github`, `.editorconfig`, `.travis.yml`
> 
This is now done. 
>     * For your _old_ experiments stuff, keep them _in_ the repo, but only local. Looks like what you did was moved it all out and just copied the files you wanted to show back into `experiments`. Do it the way I said during the meeting. Name your _old_ `experiments` folder something else (e.g. `local`), add that to the gitignore, and then use the _new_ `experiments` folder to store the specified outputs. You don't want _anything_ project-related sitting outside your repo, even if it's just local files that github will never see. Keep them all in one place.
> 
This doesn't make sense to me. I moved the files in experiments to a work_book directory within the repo that is already ignored by git. Then, after cleaning up the nbs, I moved them to the experiments directory. 
Is this not the same thing? I still have the local copy of things I don't want shown in github being ignored, but are still part of the project directory.
>     * It also wouldn't be a bad idea to rename the branch here to something more appropriate, given the name isn't exactly reflective of the fact that this branch is really now cleaning up the repo for later development.
Looking into this now.
